pdfs/src/Pdf/FromLatex.hs

```diff
@@ -35,20 +37,27 @@ import Dscp.Core.Foundation.Educator
 
 import Pdf.MkLatex
 
--- | Generate latex certificate from locate and data.
-generate :: Language -> CertificateFullInfo -> Text
-generate lang cert = do
-    Text.toStrict $ Text.toLazyText $ make (lang, cert)
+-- | Datatype containing information about Educator which is
+-- required in order to render a certificate.
+data EducatorInfo = EducatorInfo
```

Comment:
Oh kek, we already have one `EducatorInfo` [here](https://github.com/DisciplinaOU/disciplina/blob/martoon%2Fdscp457-bump/educator/src/Dscp/Educator/Web/Educator/Types.hs#L80).
Could you rename that to `EducatorStatusInfo` (to comply with endpoint name)?

Reply:
Which endpoint name, exactly?....
Better to make it `CertificateIssuerInfo` then, frankly.

Reply:
`api/educator/v1/status` endpoint, but do as you see fit.